examples/data-sources/scope/data.tf

```
@@ -0,0 +1,9 @@
+data "boundary_scope" "org" {
+  name     = "SecOps"
+  scope_id = "global"
```

Should this be `parent_scope_id` now?
```suggestion
  parent_scope_id = "global"
```